### PR TITLE
Add new Silverfish Tiers to P1

### PR DESCRIPTION
**Pull request by Hiinky**

### CODE REVIEW

File: data/base-hunting.yaml [hunk 1]
```diff
@@ -407,6 +407,31 @@ hunting_zones:
   - 1446
   - 1447
   - 1448
+  silverfish:                                                             
+  # https://elanthipedia.play.net/Silverfish                              10-32
+  # Do Not Use 6079.  It is a terrible quadruplicate.
```
Comment (jandersson) [inline data/base-hunting.yaml:412]: What is a terrible quadrublicate?

Comment (Hiinky) [reply data/base-hunting.yaml:412]: theres four rooms that are dups in mapdb and cannot be mapped better than what I did.  should never hunt there.  it's also noted on a new map image i have pending

Comment (jandersson) [reply data/base-hunting.yaml:412]: Just say why? Also this is a data file, it feels wierd to leave notes like this in my opinion

Comment (Hiinky) [reply data/base-hunting.yaml]: Updated

**Pull request merged**